examples/joy-ui-nextjs/src/components/ThemeRegistry/theme.ts

```diff
@@ -0,0 +1,45 @@
+import { extendTheme } from '@mui/joy/styles';
+import { Inter, Source_Code_Pro } from 'next/font/google';
+
+const inter = Inter({
+  subsets: ['latin'],
+  adjustFontFallback: false, // prevent NextJS from adding its own fallback font
+  fallback: ['var(--joy-fontFamily-fallback)'], // use Joy UI's fallback font
+});
+
+const sourceCodePro = Source_Code_Pro({
+  subsets: ['latin'],
+  adjustFontFallback: false, // prevent NextJS from adding its own fallback font
```

Comment:
```suggestion
  adjustFontFallback: false, // prevent Next.js from adding its own fallback font
```

Reply:
I guess both (a) and (b) have their own benefits and drawbacks. Let's say we use (b), here are some considerations.
- Less layout shift compared to "traditional" font fallback (it still shifts a little in certain cases).
- The fallback is hard coded to Arial, which may not be the best fallback for some fonts.
- `--joy-fontFamily-fallback` is essentially useless in this case, which can be confusing to some users.

I do think `const inter = Inter({ subset: ["latin"] })` is cleaner. But I guess if we decided to go this direction, we should add a comment mentioning the `--joy-fontFamily-fallback` situation to avoid any confusion.

As for the `ThemeRegistry` changes. I guess some other users are updating the Next.js example. I'm pretty sure that there used to be a `.tsx` file here.